app/assets/stylesheets/applications.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Applications controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
Lets not track these files into the git repo:
`git rm app/assets/javascripts/applications.scss`